packages/ava/src/data/utils/isType/isDateString.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Check whether the string is a date.
+ */
+
+const delimiter = '([-_./\\s])';
+const year = '(18|19|20)\\d{2}';
+const month = '(0?[1-9]|1[012])';
+const day = '(0?[1-9]|[12]\\d|3[01])';
+const week = '([0-4]\\d|5[0-2])';
+const weekday = '([1-7])';
+const hour = '(0?\\d|1\\d|2[0-4])';
+const minute = '(0?\\d|[012345]\\d)';
+const second = minute;
+const millisecond = '\\d{1,4}';
+const yearday = '((([0-2]\\d|3[0-5])\\d)|36[0-6])';
+const offset = `(Z|[+-]${hour}(:${minute})?)`;
+
+/**
+ * Get ISO 8601 date array
+ * Reference: https://www.cl.cam.ac.uk/~mgk25/iso-time.html
+ * @param strict Require delimiter or not
+ */
+export function getIsoDates(strict = true) {
```

Comment:
why export getIsoXXX ?

and these names are poorly readable.. getIsoDates? should be getIsoDatePatterns or getIsoDateRegExps?

Reply:
for test, and plan to export to users